src/App.css

```diff
@@ -1,86 +1,93 @@
 
 @import url('https://fonts.googleapis.com/css2?family=Edu+AU+VIC+WA+NT+Hand:wght@400..700&display=swap');
 
-/* header */
-header {
-  background-color: rgba(0, 0, 0, 0.744); 
-  color: white; 
-  padding: 10px 20px; 
-  position: fixed; 
+html,
+body {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  background-color: #000000;
+}
+
+.app {
+  scroll-behavior: smooth;
+}
+
+/* Header */
+.header {
+  background-color: rgba(0, 0, 0, 0.744);
+  color: white;
+  padding: 10px 20px;
+  position: fixed;
   top: 0;
   left: 0;
-  width: 100%; 
+  width: 100%;
   z-index: 1000;
-  height: 40px;
+  display: flex;
+  align-items: center; 
+ 
 }
 
 nav {
-  display: flex; 
-  align-items: center;
-  justify-content: flex-start; 
+  width: 100%;
 }
 
 ul {
-  list-style: none; 
-  padding: 0;
+  list-style: none;
   margin: 0;
+  padding: 0;
   display: flex; 
+  justify-content: flex-start; 
+  gap: 20px; 
 }
 
 li {
-  margin-right: 1px; 
+  margin: 0;
 }
 
 a {
-  text-decoration: none; 
-  color: rgb(248, 245, 245); 
+  text-decoration: none;
+  color: rgb(248, 245, 245);
   font-family: "Edu AU VIC WA NT Hand", cursive;
-  font-size: 16px; 
-  text-transform: uppercase; 
-  letter-spacing: 1px; 
-  display: inline-block; 
-  padding: 8px 16px; 
-  transition: background-color 0.3s ease; 
+  font-size: 16px;
+  text-transform: uppercase;
+  letter-spacing: 1px;
+  display: inline-block;
+  padding: 8px 16px;
+  transition: background-color 0.3s ease;
 }
 
 a:hover {
   background-color: rgba(255, 255, 255, 0.2);
- 
+  border-radius: 4px;
 }
 
-/* home page */
+/* Home Page */
 .home-page {
```

Comment:
Please create separate css file for Home  component

Reply:
fixed